### DISCUSSION TITLE
Is it possible to treat undefined as legitimate default value for Controller

### DISCUSSION THREAD

**Discussion by ajaybeniwal** [react-hook-form/react-hook-form#3547] [Q&A]
Currently we have a problem that Controller does not treat undefined as legitimate default value and keeps complaining that default value is not set which logically makes sense since we can set empty string value but that causes problem with some placeholder control  of some version of old third party library which we use , so just a question is somehow possible to treat undefined as legitimate default value for controller using some setting ?

**Comment by bluebill1049** [comment]
not really, it may work with some cases, but you will get the weird issue for others. we had an issue around this even when something wrong happened in prod because it worked in dev, not prod (due to that).
